msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -10,15 +10,55 @@ ms.author: v-npaladugu
 
 A configuration page is a special type of [content page](content-page.md). The users configure some aspects of the Microsoft Teams app using the configuration page and use that configuration as part of the following:
 
-* A channel or group chat tab: Collect information from the users and set the `contentUrl` of the content page to be displayed.
+* A personal, channel, or group chat tab: Collect information from the users and set the `contentUrl` of the content page to be displayed.
 * A [message extension](~/messaging-extensions/what-are-messaging-extensions.md).
 * A [connector for Microsoft 365 Groups](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md).
 
 [!INCLUDE [sdk-include](~/includes/sdk-include.md)]
 
-## Configure a channel or group chat tab
+To add customizable experience to your personal (static) tab, add your [configuration logic](#configuration-page-for-tabs) to your `contentUrl` code space. If you are [migrating your configurable tab to personal (static) tab](~/tabs/how-to/create-channel-group-tab.md#migrate-your-configurable-tab-to-personal-static-tab) move your configuration logic from `configurationUrl` code space to `contentURL` code space.
```

Comment:
I'm of the opinion that we should revert everything from line 19-57 back to what we had before. The reason for that is because the purpose of this page doc is _specifically_ to talk about the `configuration dialog`. I worry that conflating the configuration dialog experience with the behavior of static tab (and the nuance that yes, some of these APIs work in a static tab too) will only confuse developers. It kind of confused me and I should know this stuff.

A lot of my guidance (on configuration) when I originally wrote the draft was geared towards static tabs. Now that I'm reading that guidance in the _same_ documentation as the `configuration dialog` behavior, it gets really confusing. It's even more confusing when you realize that most static tabs (in chats or meetings) won't need to do any kind of configuration/setup (like YouTube, Forms, Polls, Polly).

We already mention elsewhere 'hey, the `setConfig` function works from your static tab` which I think is enough.